legend-pure-runtime/legend-pure-runtime-java-engine-compiled/src/main/java/org/finos/legend/pure/runtime/java/compiled/generation/processors/type/_class/ClassImplProcessor.java

```diff
@@ -1004,43 +1005,22 @@ private static String getterOverrides(String classNamePlusTypeParams)
                             CoreInstance definition = Instance.getValueForMetaPropertyToOneResolved(constraint, M3Properties.functionDefinition, processorSupport);
-                            String eval = "(Boolean) " + ValueSpecificationProcessor.createFunctionForLambda(constraint, definition, registerLambdas, processorSupport, processorContext) + ".execute(" + (extraValues == null ? "Lists.mutable.with(this)" : extraValues) + ",es)";
-
-                            String messageJavaFunction = null;
-                            CoreInstance message = Instance.getValueForMetaPropertyToOneResolved(constraint, M3Properties.messageFunction, processorSupport);
-                            if (message != null)
-                            {
-                                messageJavaFunction = ValueSpecificationProcessor.createFunctionForLambda(constraint, message, registerLambdas, processorSupport, processorContext);
-                            }
-
-                            if (owner == null || "Global".equals(owner.getName()))
+                            CoreInstance owner = Instance.getValueForMetaPropertyToOneResolved(constraint, M3Properties.owner, processorSupport);
+                           if (owner == null || "Global".equals(owner.getName()))
                             {
-                                CoreInstance expression = Instance.getValueForMetaPropertyToOneResolved(definition, M3Properties.expressionSequence, processorSupport);
-                                CoreInstance constraintClass = Instance.getValueForMetaPropertyToOneResolved(expression, M3Properties.usageContext, processorSupport).getValueForMetaPropertyToOne(M3Properties.type);
-
-                                String errorMessage = message == null ?
-                                        "\"Constraint :[" + ruleId + "] violated in the Class " + constraintClass.getValueForMetaPropertyToOne(M3Properties.name).getName() + "\"" :
-                                        "\"Constraint :[" + ruleId + "] violated in the Class " + constraintClass.getValueForMetaPropertyToOne(M3Properties.name).getName() + ", Message: \" + (String) " + messageJavaFunction + ".execute(" + (extraValues == null ? "Lists.mutable.with(this)" : extraValues) + ",es)";
-
-                                return
-                                        "            if (!(" + eval + "))\n" +
-                                                "            {\n" +
-                                                "                throw new org.finos.legend.pure.m3.exception.PureExecutionException(sourceInformation, " + errorMessage + ");\n" +
-                                                "            }\n";
-                            }
+                                validateItems.accumulateAndGet(validateItem(stateAndDeep, constraint, _class, processorContext, extraValues) + "\n",
+                                        (existing, newValue) -> existing + newValue);
```

Comment:
If you change validateItems to a StringBuilder, as I suggested above, this would become:
```suggestion
                                validateItems.append(validateItem(stateAndDeep, constraint, _class, processorContext, extraValues)).append("\n");
```